src/walletSdk/Watcher/index.ts

```diff
@@ -296,12 +294,13 @@ export class Watcher {
         const registeredTransaction =
           this._transactionsRegistry[assetCode][transaction.id];
 
-        // if we've had the transaction before, only report if there is a change
+        // if we've had the transaction before, only report if there is a status change
+        let isChanged = true;
```

Comment:
in doing this found that the watcher for watching one transaction will stop if we get the same transaction twice

spoke with @CassioMG and agreed it should change to continue watching